wolfcrypt/src/asn.c

```diff
@@ -9461,7 +9461,12 @@ static char GetNameType(CertName* name, int idx)
 #ifdef WOLFSSL_CERT_EXT
     case 7:
         return name->busCatEnc;
+
+    case 8:
```

Comment:
Why was this approved but not merged??  This needs to be documented, why are there empty case 7 and 8 that don't return a valid a type?

Reply:
The empty case is to keep index parity with the other name methods, ``GetOneName``` and ```GetNameId```. The original code simply omits the email OID encoding string type. But when a new name OID was handled, the email OID index is still the last. I added a comment to help clarify the reason for the empty case.